src/Stratis.Features.Collateral/JoinFederationRequestMonitor.cs

```diff
@@ -129,8 +129,10 @@ public void OnBlockConnected(VotingManagerProcessBlock blockConnectedData)
                             this.votingManager.CreatePendingPoll(blockConnectedData.PollsRepositoryTransaction, votingData, blockConnectedData.ConnectedBlock.ChainedHeader);
                         }
                     }
-
-                    this.votingManager.ScheduleVote(votingData);
+                    
+                    // If this node is a federation member then schedule a vote.
+                    if (this.votingManager.IsFederationMember)
```

Comment:
Should we then use `IsMemberOfFederation` here?

Reply:
That is implicit.